### ISSUE TITLE
Color visualization selector svgs based on theme

### ISSUE THREAD

**Issue by asizemore** [VEuPathDB/web-eda#1255]
Visualization selector svgs were created with the eventual idea of incorporating theme colors. That is, they already have a primary and secondary color organization. Currently these colors are hard coded but they could be somehow passed to the svg?

Then in each visualization component, when creating `SelectorComponent` we could ask if there's a theme color, apply it if yes, and if not use default dark and light gray (or some other default scheme).

**Comment by SheenaTomko** [comment]
@chowington is this why the colors here look so much more intense? Are these the final primary and secondary color selections?

**New**
<img width="1272" alt="Screen Shot 2022-10-10 at 4 04 14 PM" src="https://user-images.githubusercontent.com/54597319/194943898-7a8fdd7c-4a4f-41a5-bec6-db270674fec4.png">


**Old**
<img width="1274" alt="Screen Shot 2022-10-10 at 4 04 31 PM" src="https://user-images.githubusercontent.com/54597319/194943934-1fafdfff-276a-449d-81e7-c9684a6b7990.png">

**Comment by chowington** [comment]
Yes, this is why the colors are different. I used the defined primary theme color for the site and chose a color level I thought was closest to what we had before. I wouldn't say they're final. The only one I thought was drastically different was the color of the dots on the map, and I couldn't quite figure out why they were so much lighter, because I wasn't giving them that light of a color.

**Comment by SheenaTomko** [comment]
Thanks Connor! That's helpful to know. 

@danicahelb, are the color levels for things like this worth discussing in either the styling meeting or the EDA UX meeting? I'm not against leaving the colors as they are now (both colors seem more vivid than before), I just don't want to close the ticket if you think others would want to provide feedback

**Comment by danicahelb** [comment]
I'm fine with the new colors. I think we have bigger things to discuss so won't add this to an agenda unless someone complains! Thanks!

**Comment by SheenaTomko** [comment]
Cool, moving this to complete
